### PR TITLE
Enable Tensorboard and save to 'model' file

### PR DESCRIPTION
**Pull request by tomasruizt**

### CODE REVIEW

File: gym_roboy/example_control.py [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
-    for _ in range(100):
-        obs = env.reset()
-        done = False
-        while not done:
-            action, _ = agent.predict(obs)
-            obs, reward, done, info = env.step(action)
-            print("step reward:", reward)
-            time.sleep(0.01)
-        print("reached goal!")
+    while True:
```
Comment (BarisYazici) [inline gym_roboy/example_control.py]: Do we need while?

File: gym_roboy/example_control.py [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
 from stable_baselines import PPO1
 
-from .envs import MsjEnv, MsjROSBridgeProxy
+from .envs import MsjEnv
 
 
-def our_env_constructor() -> gym.Env:
-    return MsjEnv(MsjROSBridgeProxy())
+MOUNT_DIR = "/root/develDeepAndReinforced/tensorboard_out/"
+MODEL_FILE = os.path.join(MOUNT_DIR, "model")
```
Comment (BarisYazici) [inline gym_roboy/example_control.py]: Let's think of a more meaningful name which involves ppo1 as well. Might be useful if we want to compare the performance of different algorithms.

File: gym_roboy/example_control.py [hunk 3]
```diff
@@ -1,34 +1,30 @@
...
     # The algorithms require a vectorized environment to run
-    env = DummyVecEnv([our_env_constructor])
-
-    agent = PPO1(MlpPolicy, env, verbose=1)
+    env_constructor = MsjEnv
+    env = DummyVecEnv([env_constructor])
 
-    agent.learn(total_timesteps=1000)
+    if os.path.isfile(MODEL_FILE):
+        agent = PPO1.load(MODEL_FILE)
```
Comment (BarisYazici) [inline gym_roboy/example_control.py]: Can the loaded agent still learn? I think stable-baseline assumes that loaded agent only be used for prediction.

File: gym_roboy/example_control.py (hunk 2 above)
Comment (tomasruizt) [reply gym_roboy/example_control.py]: sure, any suggestion? What about 
```python
MODEL_FILE = os.path.join(MOUNT_DIR, "2019_02_03_PPO1")
```

File: gym_roboy/example_control.py (hunk 1 above)
Comment (tomasruizt) [reply gym_roboy/example_control.py]: with the `while True` the agent will keep learning and saving at intervals. If we want to stop it, we CTRL+C

File: gym_roboy/example_control.py (hunk 3 above)
Comment (tomasruizt) [reply gym_roboy/example_control.py]: I think it should, but if you want, I can write a test for it. Shall I?

File: gym_roboy/example_control.py (hunk 2 above)
Comment (tomasruizt) [reply gym_roboy/example_control.py]: it could also involve something like `str(datetime.now())`

Comment (BarisYazici) [reply gym_roboy/example_control.py]: ok

File: gym_roboy/example_control.py (hunk 3 above)
Comment (BarisYazici) [reply gym_roboy/example_control.py]: I tested. just add the line `agent.set_env(env)`

Comment (tomasruizt) [reply gym_roboy/example_control.py]: is that really required? The loaded agent does learn, which means that it knows its environment.

Comment (tomasruizt) [reply gym_roboy/example_control.py]: I'll change the loading logic as discussed

**Comment by BarisYazici** [comment]
If we decide to use this agent we can remove other agent in other repo

**Comment by tomasruizt** [comment]
@BarisYazici I think as a next step, we can move this file to "deep_roboy_contol"

**Pull request merged**